scripts/scr_special_view/scr_special_view.gml

```diff
@@ -33,50 +33,37 @@ function scr_special_view(argument0) {
 
 
 
-
+	var non_marine_roles = ["Skitarii","Crusader","Sister of Battle","Sister Hospitaler","Ork Sniper","Flash Git"]
```

Comment:
don't we have this defined elsewhere? does it make sense to copy-paste this each time, instead of just reading the values from one, common source?

Reply:
yeah i could look around and try and aggregate them ultimately though i have a way of avoiding this method anyway so probably sit tight then i'll do that refactor